src/formatter/formatPropValue.js

```diff
@@ -53,6 +53,14 @@ const formatPropValue = (
     )}}`;
   }
 
+  // handle forwardRef and memo
+  if (isPlainObject(propValue) && propValue.$$typeof) {
```

Comment:
I tried to use `react-is` values here but it didn't seem to work for some reason.

Reply:
`react-is` is mainly designed to test **element** types but you want to check if a component is of a particular type.

You can still do it, although in a somewhat clunky way:

```suggestion
  if (isPlainObject(propValue) && (propValue.$$typeof === Memo || propValue.$$typeof === ForwardRef)) {
```